src/trrntzip.c

```diff
@@ -160,8 +168,29 @@ void FreeWorkspace(WORKSPACE *ws) {
   free(ws);
 }
 
-// Get the filelist from the zip file in canonical order
-// Returns a sorted array
+// The canonical order is case insensitive, but we need a tie-breaker
+// to avoid ambiguity
+static int CanonicalCmp(const char *s1, const char *s2) {
```

Comment:
These functions could be in `util.c` like one of them was before, any particular reason for moving them?

Reply:
Not really, I just thought the comparison functions for `qsort()` don't feel like general utilities. I can move them back to `util.c` if you prefer.

Reply:
Yes, please.